### PR TITLE
Governance sub menu

### PR DESCRIPTION
**Pull request by msavir**
Desktop

image

Mobile

image

image

### CODE REVIEW

File: src/components/Nav.tsx [hunk 1]
```diff
@@ -38,6 +57,22 @@ export default function Nav() {
...
+        anchorEl={governanceMenuEl}
+        TransitionComponent={Fade}
+        aria-controls={open ? "fade-menu" : undefined}
+        aria-haspopup="true"
+        aria-expanded={open ? "true" : undefined}
+      >
+        <MenuItem onClick={() => handleCloseAndNavigate("/proposals")}>
+          Proposals
+        </MenuItem>
+        <MenuItem onClick={() => handleCloseAndNavigate("/proposals/stake")}>
```
Comment (zihan-aptos) [inline src/components/Nav.tsx:72]: Have we decided on the name "Staking"? If so, I would change the route to `/proposals/staking`.

**Review by zihan-aptos — COMMENTED** [review]
Looks good! I'll leave it to @moonclavedev

**Comment by moonclavedev** [comment]
> Desktop
> 
> image
> 
> Mobile
> 
> image

Overall it's good but we will have to do some work on managing the text hierarchy here as it's a little awkward on mobile - this is more obvious because we don't have a top-level toggle with the arrow on this like we do on the marketing site. 

Maybe a quick fix for now would be to put a down arrow on the right as if it were already expanded? 

image

**Comment by msavir** [comment]
> > Desktop
> > image
> > Mobile
> > image
> 
> Overall it's good but we will have to do some work on managing the text hierarchy here as it's a little awkward on mobile - this is more obvious because we don't have a top-level toggle with the arrow on this like we do on the marketing site.
> 
> Maybe a quick fix for now would be to put an arrow on the right as if it were already expanded? This does break the rules and users might be confused as to why they can't toggle it however.
> 
> image

@moonclavedev great idea!

something like that?

image

image

**Comment by moonclavedev** [comment]
Also, can we change "Stake" to "Staking"? @CapCap

**Comment by moonclavedev** [comment]
> > > Desktop
> > > image
> > > Mobile
> > > image
> > 
> > 
> > Overall it's good but we will have to do some work on managing the text hierarchy here as it's a little awkward on mobile - this is more obvious because we don't have a top-level toggle with the arrow on this like we do on the marketing site.
> > Maybe a quick fix for now would be to put an arrow on the right as if it were already expanded? This does break the rules and users might be confused as to why they can't toggle it however.
> > image
> 
> @moonclavedev great idea!
> 
> something like that?
> 
> image
> 
> image

Yep!

**Review by moonclavedev — APPROVED** [review]

**Pull request merged**